### PR TITLE
Update onboarding doc to use cli.

### PR DESCRIPTION
**Pull request by HumairAK**
Related: https://github.com/operate-first/apps/issues/760

Also removed the use of `kustomize edit add` so we can have people add it alphabetically (also avoid reformatting the kustomization making the diff unnecessarily large).

### CODE REVIEW

**Review by 4n4nd — COMMENTED** [review]
@HumairAK can you also add that once all the files have been generated, the user needs to create a PR to the original repo and the changes will be applied once that PR is approved and merged.
I have had people assuming that the onboarding script applies the changes for them, which is understandable.

**Comment by HumairAK** [comment]
@4n4nd we mention making a PR for the changes, do you recommend we move this higher up and rephrase this to be a bit more clear?

**Comment by 4n4nd** [comment]
yeah I would say somewhere before we ask the user to log in to the cluster ([here](https://github.com/operate-first/support/blob/910f77a9843c1a6d82cab797b1490631832a2363/docs/onboarding_to_cluster.md#authenticate-via-openshift))?

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Comment by HumairAK** [comment]
@4n4nd I added it [here](https://github.com/operate-first/support/pull/273/files#diff-3fc0513183fd1a4dbc16de3061bb0a0c782b7c96ea3380f458d7e4afc64d910bR10), it made more sense to me -- wdyt?
